docs/UserGuide.md

```diff
@@ -296,31 +296,21 @@ Each candidate is uniquely identified by the same name, email and phone number.
 Manage a list of interviews for you to select the desired candidates, with the simple instructions below!
 
 #### <u>Add an interview:</u> `add_i`
+Use the following command to record the details of an interview session with the candidate(s) for a position!
 
 *Adds an interview to the list of interviews.*
 
 <u>Format:</u>
 
-    add_i position=<POSITION> index=<INDEX>... date=DATE time=TIME duration=DURATION [interviewed=STATUS]
+    add_i position=<POSITION> [c=<INDEX>]... date=DATE time=TIME duration=DURATION [interviewed=STATUS]
```

Comment:
Pretty sure c= is a compulsory field

Reply:
Because I caught "c=" works without any numbers, so I thought maybe it can be optional entirely since we have assign and unassign, but havent got to change the actual code for add_i yet. So should we allow index to be optional or should it be compulsory then?

Reply:
I think we can make it optional. And need to change the code for add_i, like make it such that the "c=" is not required